src/PageContainer.js

```diff
@@ -13,42 +15,51 @@ class PageContainer extends Component {
         super(props);
 
         this.state = {
-            content: props.content
+            content: props.content,
+            account: props.account
         };
+
+        this.navBarHandler = this.navBarHandler.bind(this)
     }
 
+    navBarHandler(e) {
+        this.setState({
+            content: e
+        })
+        console.log('container: ' + this.state.content)
+    }
+ 
     render(){
         return(
-            <div className="">
-                <header className="">
-                    <PageHeader title={this.props.account.name}></PageHeader>
-                </header>
-                <NavBar accountType={this.props.account.accountType}></NavBar>
+            <div>
+                {/* <header > */}
+                    <PageHeader logo={logo} title={this.props.account.name}></PageHeader>
+                {/* </header> */}
+                <NavBar content={this.state.content} accountType={this.props.account.accountType} handler={this.navBarHandler.bind(this)}></NavBar>
```

Comment:
you don't need to bind the `navBarHandler` to `this` again here, you already did it in constructor :) I like that binding is done in constructor so it's only done once.